pkgs/by-name/fl/flatito/package.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, fetchFromGitHub, ruby, buildRubyGem, bundlerEnv }:
+let
+  deps = bundlerEnv rec {
+    name = "flatito-${version}";
+    source.sha256 = lib.fakeSha256;
+    version = "0.1.1";
+    inherit ruby;
+    gemdir = ./.;
+    gemset = lib.recursiveUpdate (import ./gemset.nix) {
+      flatito.source = {
+        remotes = [ "https://rubygems.org" ];
+        sha256 = "9f5a8f899a14c1a0fe74cb89288f24ddc47bd5d83ac88ac8023d19b056ecb50f";
+        type = "gem";
+      };
+    };
+  };
+in
+
+buildRubyGem rec {
+  inherit ruby;
+
+  gemName = "flatito";
+  pname = gemName;
+  version = "0.1.1";
+
+  source.sha256 = "sha256-n1qPiZoUwaD+dMuJKI8k3cR71dg6yIrIAj0ZsFbstQ8=";
```

Comment:
Can't we use `source.hash` ?

Reply:
We are trying to update the automatically-generated gemset.nix flatito sha256, and this is how ruby's distribution handles it. 

https://github.com/Rucadi/nixpkgs/tree/flatito/pkgs/by-name/fl/flatito/gemset.nix#L22-L31